proto/src/determined/experiment/v1/experiment.proto

```diff
@@ -36,6 +36,10 @@ enum State {
   STATE_DELETE_FAILED = 11;
   // The experiment is killed and is shutting down.
   STATE_STOPPING_KILLED = 12;
+  // The experiment has an allocation which is being spun up for use.
+  STATE_PENDING = 13;
```

Comment:
✅ - we might talk about this later, but does it work for Pending and Queued to be substates of Active, and then when the API returns Active it means the experiment is running?

Reply:
maybe we talked about it, i think i'd prefer active -> running everywhere, otherwise the states are a bit confusing (active implies pending/queued/running is less confusing than active imples pending/queued/active because its less clear what active actually is)